api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -173,33 +187,31 @@ public ModelAndView searchJsonPost(
      * @return the JSON response
      */
     @ApiOperation(value = "search for records", nickname = "searchRecords", response = Void.class)
-    @GetMapping(value = {
-            "/api/v2/search.json",
-            "/record/v2/search.json",
-            "/record/search.json"
-    }, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ModelAndView searchJsonGet(
-            @RequestParam(value = "wskey") String apikey,
-            @SolrEscape  @RequestParam(value = "query") String queryString,
-            @RequestParam(value = "qf", required = false) String[] refinementArray,
-            @RequestParam(value = "reusability", required = false) String[] reusabilityArray,
-            @RequestParam(value = "profile", required = false, defaultValue = "standard") String profile,
-            @RequestParam(value = "start", required = false, defaultValue = "1") int start,
-            @RequestParam(value = "rows", required = false, defaultValue = "12") int rows,
-            @RequestParam(value = "facet", required = false) String[] mixedFacetArray,
-            @RequestParam(value = "theme", required = false) String theme,
-            @RequestParam(value = "sort", required = false) String sort,
-            @RequestParam(value = "colourpalette", required = false) String[] colourPaletteArray,
-            @RequestParam(value = "thumbnail", required = false) Boolean thumbnail,
-            @RequestParam(value = "media", required = false) Boolean media,
-            @RequestParam(value = "text_fulltext", required = false) Boolean fullText,
-            @RequestParam(value = "landingpage", required = false) Boolean landingPage,
-            @RequestParam(value = "cursor", required = false) String cursorMark,
-            @RequestParam(value = "callback", required = false) String callback,
-            @SolrEscape @RequestParam(value = "hit.fl", required = false) String hlFl,
-            @RequestParam(value = "hit.selectors", required = false) String hlSelectors,
-            HttpServletRequest request,
-            HttpServletResponse response) throws EuropeanaException {
+    @GetMapping(value = {"/api/v2/search.json", "/v2/search.json", "/record/v2/search.json", "/record/search.json"},
```

Comment:
"/v2/search.json" is not a endpoint we want to advertise

Reply:
see above